modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -182,7 +182,9 @@
 					_this.pagingInProgress = true;
 					_this.page++; // move to next page
 					mw.log("Playlist:: scrollEnd event. paging: true: trying to load next playlist page: page="+_this.page+", pageSize"+_this.getConfig('pageSize'));
-					var playlistRequest = {
+
```

Comment:
@vladm3 - for readability and prevent redundant $.extend call, please separate the 2 requests creation:

if ( _this.playlistSet[_this.currentPlaylistIndex].playlistParams){
// use saved params and use $.extend
}else{
// create the original request
}